samigo/samigo-app/src/webapp/css/tool_sam.css

```diff
@@ -1060,4 +1060,93 @@ ul.no-list {
 td.feedbackColumn2 {
   vertical-align: bottom;
   padding-bottom: 15px;
+}
+
+#questionProgressPanel
+{
+	min-height: 200px;
+	max-height:auto;
+	width: 295px;
+	background: #ddd;
```

Comment:
The file is internally inconsistent. And I can't tell at a glance which is more dominant, tabs or spaces.